src/components/Pagination/Pagination.js

```diff
@@ -1,9 +1,28 @@
-import React from "react";
 import { Pagination } from "antd";
-function PaginationApp() {
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { changePage } from "redux/productsSlice";
+
+function PaginationApp(props) {
+  const dispatch = useDispatch();
+  const filters = useSelector((state) => state.products.filters);
+  const onChange = (current) => {
```

Comment:
Tránh đặt tên func là on ở đầu nhé, dễ nhầm lẫn với các event onChange, onSubmit của các thẻ
```suggestion
  const handleChangePage = (current) => {
```